### PR TITLE
refactor CONTRIBUTING.md into multiple files to help readability

### PR DESCRIPTION
**Pull request by ultrasaurus**
refactoring in advance of adding more content for https://github.com/cncf/tag-security/issues/613

### CODE REVIEW

File: CONTRIBUTING/writing-style.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Writing style
```
Comment (lumjjb) [inline CONTRIBUTING/writing-style.md:1]: There is no reference to this file from any of the other files, probably will be helpful to link from the contributing README.md

File: CONTRIBUTING/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Contributing
```
Comment (lumjjb) [inline CONTRIBUTING/README.md:1]: Would be nice to have this be linked from main README..

**Review by lumjjb — APPROVED** [review]
LGTM, with minor nits! Awesome, thanks @ultrasaurus.... there's one issue that I'd like to get some feedback on: should the directory be UPPERCASE or lowercase, I can see arguments for both, either more visibility, or for consistency... Or maybe add a pointer from a shell CONTRIBUTING.md

File: CONTRIBUTING/writing-style.md (hunk 1 above)
Comment (ultrasaurus) [reply CONTRIBUTING/writing-style.md:1]: fixed

File: CONTRIBUTING/README.md (hunk 2 above)
Comment (ultrasaurus) [reply CONTRIBUTING/README.md:1]: agree, let's make that a separate PR

**Comment by ultrasaurus** [comment]
I'm going to merge now.  Wanted to keep all upper case for consistency with CONTRIBUTING.md.  Going to merge now (big fan of many small changes to existing docs, when possible).  Wouldn't be against changing name of directory, but wanted to see if this gets picked up automatically by github or if we need to link from .github/

**Pull request merged**